chashmap/src/lib.rs

```diff
@@ -242,9 +242,7 @@ impl<K: PartialEq + Hash, V> Table<K, V> {
                 return lock;
             }
         }
-
-        // TODO
-        unreachable!();
+	panic!("scan failed! no entry found");
```

Comment:
Maybe also change the comment to:

    `CHashMap` scan failed! No entry found.